runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/resolver/ExpressionResolverSpi.java

```diff
@@ -0,0 +1,9 @@
+package io.aklivity.zilla.runtime.engine.resolver;
```

Comment:
Please refactor package name to `io.aklivity.zilla.runtime.engine.expression` instead.